src/plugin.js

```diff
@@ -309,9 +324,17 @@ const plugin = function(options) {
   // because it doesn't make sense to pass it to each Overlay component.
   delete settings.overlays;
 
-  this.overlays_ = overlays.map(
-    o => this.addChild('overlay', videojs.mergeOptions(settings, o))
-  );
+  this.overlays_ = overlays.map(o => {
+    let mergeOptions = videojs.mergeOptions(settings, o);
+
+    // Attach bottom aligned overlays to the control bar so
+    // they will adjust positioning when the control bar minimizes
+    if (this.controlBar && mergeOptions.align.indexOf('bottom') !== -1) {
```

Comment:
Part of my hesitation with doing it purely by CSS is that I want the movement of the overlay to be in sync with the animation of the control bar moving down. If I did this in CSS I would have to assume a certain type of minimizing animation which would really only work out of the box for one particular skin.

I'm certainly open to giving the idea more thought though.